examples/va/nfv/ovs-dpdk-sriov/edpm/values.yaml

```diff
@@ -0,0 +1,203 @@
+    # The public key that will have access to the dataplane computes via SSH
+    public: CHANGEME3
+  nodeset:
+    ansible:
+      ansibleUser: cloud-admin
+      ansiblePort: 22
+      ansibleVars:
+        # CHANGEME -- see https://access.redhat.com/solutions/253273
+        # edpm_bootstrap_command: |
+        #       subscription-manager register --username <subscription_manager_username> --password <subscription_manager_password>
+        #       podman login -u <registry_username> -p <registry_password> registry.redhat.io
+        service_net_map:
+          nova_api_network: internalapi
+          nova_libvirt_network: internalapi
+        timesync_ntp_servers:
+          - hostname: clock.redhat.com
+        # CPU pinning settings
+        # edpm nfv ovs dpdk config
+        # CHANGEME
+        edpm_kernel_args: "default_hugepagesz=1GB hugepagesz=1G hugepages=64 iommu=pt intel_iommu=on tsx=off isolcpus=2-11,14-23"
+        edpm_tuned_profile: "cpu-partitioning-powersave"
+        edpm_tuned_isolated_cores: "2-11,14-23"
+        edpm_nova_libvirt_qemu_group: "hugetlbfs"
+        edpm_ovs_dpdk_pmd_core_list: "1,13,2,14,3,15"
+        edpm_ovs_dpdk_socket_memory: "4096"
+        edpm_ovs_dpdk_memory_channels: "4"
+        edpm_ovs_dpdk_vhost_postcopy_support: "true"
+        edpm_ovn_bridge_mappings: ['dpdk2:br-link2','dpdk1:br-link1']
+        # edpm nfv sriov config
+        edpm_neutron_sriov_agent_SRIOV_NIC_physical_device_mappings: 'sriov1:eno5,sriov2:eno6'
+        # edpm_network_config
+        # These vars are edpm_network_config role vars
+        edpm_network_config_hide_sensitive_logs: false
+        edpm_network_config_os_net_config_mappings:
+          # Need to provide nic mapping based on system uuid or system product name
+          # here used systetm uuid
+          edpm-compute-0: # CHANGEME
+            dmiString: "system-uuid" # CHANGEME
+            id: "4c4c4544-0034-3810-8050-cac04f325432" # CHANGEME
+            nic1: eno1 # CHANGEME
+            nic2: eno2 # CHANGEME
+            nic3: eno3 # CHANGEME
+            nic4: eno4 # CHANGEME
+            nic5: eno5 # CHANGEME
+            nic6: eno6 # CHANGEME
+        edpm_network_config_template: |
+          ---
+          {% set mtu_list = [ctlplane_mtu] %}
+          {% for network in role_networks %}
+          {{ mtu_list.append(lookup('vars', networks_lower[network] ~ '_mtu')) }}
+          {%- endfor %}
+          {% set min_viable_mtu = mtu_list | max %}
+          network_config:
+          - type: ovs_bridge
+            name: {{ neutron_physical_bridge_name }}
+            mtu: {{ min_viable_mtu }}
+            use_dhcp: false
+            dns_servers: {{ ctlplane_dns_nameservers }}
+            domain: {{ dns_search_domains }}
+            addresses:
+            - ip_netmask: {{ ctlplane_ip }}/{{ ctlplane_cidr }}
+            routes: {{ ctlplane_host_routes }}
+            members:
+            - type: interface
+              name: eno2
```

Comment:
Maybe since relative mappings are used below, we should use the relative name here too?

Reply:
@abays yes, will update it. thanks.